src/nuklear_button.c

```diff
@@ -100,6 +100,8 @@ nk_draw_button(struct nk_command_buffer *out,
 
     if (background->type == NK_STYLE_ITEM_IMAGE) {
         nk_draw_image(out, *bounds, &background->data.image, nk_white);
+    }  else if (background->type == NK_STYLE_ITEM_9SLICE) {
```

Comment:
Have you considered using *switch-case* instead of *if-else-if-else-...* construct? Understandability and performance could be better on bad compilers (e.g. those for embedded systems which Nuklear is also targetting).

Reply:
As there are only 3 conditional checks, I didn't feel that it was necessary to add verbosity and to force the compiler to generate a jump table. If you would prefer to use a switch-case, though, I can convert it to one.

Reply:
It's a nitpick, so I'll leave it up to you. The rule of thumb I follow is "if there is just one expression to evaluate, then don't copy it N-times, but use switch( expr ) instead".

Reply:
I replaced them with switch-case as I would rather adhere to your coding style.